.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Build
+
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: docker build -t apacheds .
+  upload:
+    needs: build
+    if: startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    env:
+      name: openmicroscopy/apacheds
+    steps:
+      - name: Get other tags
+        id: gettags
+        uses: jupyterhub/action-major-minor-tag-calculator@v2.0.0
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          prefix: "${{ env.name}}:"
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.name }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Push to Docker Hub
```

Comment:
We had a problem with the last release of openmicroscopy.
The Push did not work and @dominikl had to build manually and pushed.
I did not have time to look into

Reply:
Ah, ok. That would be unfortunate. Happy to work on it via this build though.

Reply:
After your merge, I created a release&tag via the GUI. Looks like everything worked here:

https://hub.docker.com/r/openmicroscopy/apacheds/tags 
![image](https://github.com/user-attachments/assets/189a6cf6-1c37-4b7c-a162-7e9c84e41461)